code/workspaces/infrastructure-api/src/stacks/nbviewerStack.js

```diff
@@ -12,12 +12,12 @@ function createNbViewerStack(params) {
 }
 
 function deleteNbViewerStack(params) {
-  const { datalabInfo, name, type } = params;
+  const { projectKey, name, type } = params;
   const k8sName = `${type}-${name}`;
```

Comment:
Could standardise this like the other functions;
i.e
```
const k8sName = nameGenerators.deploymentName(name, type);
```

Reply:
Good idea, done.